src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -573,6 +601,20 @@ void BLEManagerImpl::_OnPlatformEvent(const ChipDeviceEvent * event)
         if (ConfigurationMgr().IsFullyProvisioned())
         {
             mFlags.Clear(Flags::kAdvertisingEnabled);
+
+            if (mIsChipoBleServiceRegistered)
```

Comment:
Should we handle that in `StopAdvertising` if number of connection is 0? I mean we should also unregister the service in case the advertising is stopped for other reasons such as the timeout.

Reply:
Added unregistering CHIPoBLE GATT service in DriveBLEState, when number of connections is equal to 0.